src/app/calculator/compressed-air/air-flow-conversion/air-flow-conversion.component.ts

```diff
@@ -63,9 +61,6 @@ export class AirFlowConversionComponent implements OnInit {
     this.airFlowConversionInputSub = this.airFlowConversionService.airFlowConversionInput.subscribe(value => {
       this.calculate();
     })
-    this.airFlowConversionOutputSub = this.airFlowConversionService.airFlowConversionOutput.subscribe(value => {
-      this.airFlowConversionOutput = value;
-    })
     this.currentFieldSub = this.airFlowConversionService.currentField.subscribe(value => {
```

Comment:
"currentField" is used for field by field help text. Changes what help text is displayed based on the current field selected. It does not appear that this calculator has field by field help text so all currentField stuff can be removed. I believe in the form there is "focusOut" as well that usually sets back to default and is not being used. Can be removed